modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition_manager.py

```diff
@@ -137,54 +135,6 @@ def get_indices(cls, axis, partitions, index_func=None):
         new_idx = ray.get(new_idx)
         return new_idx[0].append(new_idx[1:]) if len(new_idx) else new_idx
 
-    @classmethod
-    def broadcast_apply(cls, axis, apply_func, left, right, other_name="r"):
```

Comment:
I wonder why there is no `progress_bar_wrapper` decorator here? Should we leave the method here and add the decorator?

Reply:
I do not know the principle by which the functions were chosen, to which this decorator is applied :)

@devin-petersohn do you know something about that?

Reply:
That is probably our omission. The decorator should be applied to all the functions of the ray partition manager.

Reply:
@YarShev I think this is a great new task to learn the current approach and identify problems. Please create separate tracker.

Reply:
#4167